mindmeld/system_entity_recognizer.py

```diff
@@ -698,6 +698,23 @@ def duckling_item_to_query_entity(query, item, offset=0):
         return
 
 
+def duckling_item_to_entity_dict(item):
+    """Converts a duckling candidate into a an entity dictionary to be used by Annotators.
+    Args:
+        item (dict): The duckling item
+
+    Returns:
+        entity (dict): Converted entity
+    """
+    return {
+        "body": item["body"],
```

Comment:
Should this be renamed to "text" to keep it consistent with our existing schema?

Reply:
Good point, removed the usage of "entity_dict". All Annotators now return QueryEntity objects from the parse() command.